ios/FluentUI/Tab Bar/TabBarView.swift

```diff
@@ -64,6 +64,10 @@ open class TabBarView: UIView {
         }
     }
 
+    @objc public func itemView(item: TabBarItem) -> UIView? {
```

Comment:
why are you creating extra function? why won't you just expose the private one?

Reply:
I tried this first but got "Method cannot be declared public because its result uses an internal type" - `class TabBarItemView` is not `public` - should we make it public it or keep this `UIView?` wrapper?